Automattic/wp-calypso
# Tiled gallery block: rename groups to columns in tiled-grid classes

#### Changes proposed in this Pull Request

* Rename `Jetpack_Tiled_Gallery_Grouper` to `TiledGalleryLayout`
* Rename "groups" to "columns" to align with changes in React components at https://github.com/Automattic/wp-calypso/pull/29559

#### Testing instructions

- Add tiled gallery block in Gutenberg editor https://calypso.live/block-editor?branch=update/tiled-gallery-groups2columns
- It should behave just like it did before:
   <img width="619" alt="screenshot 2018-12-19 at 15 35 05" src="https://user-images.githubusercontent.com/87168/50224001-5d38c700-03a5-11e9-9c7c-8fcc0b063cba.png">
